dms/models/dms_access_groups.py

```diff
@@ -19,6 +20,7 @@ class DmsAccessGroups(models.Model):
     perm_create = fields.Boolean(string="Create Access")
     perm_write = fields.Boolean(string="Write Access")
     perm_unlink = fields.Boolean(string="Unlink Access")
+    perm_read = fields.Boolean(string="Read Access", default=True)
```

Comment:
This permission was removed several versions ago because it was redundant, i.e. none of these options would make sense:
- Create an access group with other permissions but no read permission.
- Create an access group without any permissions (read, create, write and delete).

Therefore, I think that this field is not necessary and any user who is in an access group, has read access (if you want to create a read-only group, you would not select any permission and it would be enough).